ansible/roles/postgresql/tasks/postgis.yml

```diff
@@ -21,7 +21,7 @@
   postgresql_db:
     db: postgis_template
     state: present
-  #sudo_user: postgres
+#  sudo_user: postgres
```

Comment:
This doesn't look like a material change and could be reversed before merging this PR.